app/emergency/page.tsx

```
@@ -6,6 +6,8 @@ import { ScrollArea } from '@/components/ui/scroll-area'
 import { motion, AnimatePresence } from 'framer-motion'
 import { Ambulance, Phone, BadgeIcon as Police, FlameIcon as Fire } from 'lucide-react'
 import { Avatar, AvatarImage, AvatarFallback } from '@/components/ui/avatar'
+import { Input } from "@/components/ui/input"
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card"
```

ESLint

[error] 'Card' is defined but never used.

(@typescript-eslint/no-unused-vars)